src/compute_horde_prompt_gen/run.py

```diff
@@ -19,7 +20,10 @@ def generate_prompts(
     max_new_tokens: int = 2000,
     temperature: float = 1.0,
     filepath: str = "prompts.txt",
+    leftover_prompts: deque = None,
```

Comment:
```suggestion
    leftover_prompts: deque | None = None,
```
Please don't hate me for this :D

Reply:
I... I... :D